.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,32 @@
+###Description of Issue/Question
+
+
+### Did you follow the steps from https://github.com/napalm-automation/napalm#faq
+[ ] Yes
```

Comment:
Apologies, without the "x" and with hyphen at the beginning of the line, was supposed to sound like:

```
- [ ] Yes
- [ ] No
```